client/components/docs-example/index.jsx

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Mixed with the other files should be fine. Basically the "section" level folders in client (my-sites, reader, me, devdocs) should be the groups where we put app-components to give a little bit of order and separation from client/components (general UI building blocks).